tests/testthat/test-fit-shared.R

```diff
@@ -126,7 +126,7 @@ test_that("cmdstan_summary() and cmdstan_diagnose() work correctly", {
     } else if (method == "generate_quantities") {
       expect_error(fit$cmdstan_summary(), "Not available for generate_quantities method")
       expect_error(fit$cmdstan_diagnose(), "Not available for generate_quantities method")
-    } else {
+    } else if (method == "sample") {
```

Comment:
stansummary with VB CSV's had a non-fatal error, but its not really intented for VB. In 2.26 it errors, so we should not test that.